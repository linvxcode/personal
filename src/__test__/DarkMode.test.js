

import React from "react";
import { render, screen } from "@testing-library/react";
import DarkMode from "../Component/DarkMode";

describe('Darkmode', () => {
    it('Render Darkmode', () => {
        const {Dark} = render(<DarkMode />);
        const text = screen.getByTestId('Darkmode');
        expect(text).toBeInTheDocument();
        expect(Dark).toMatchSnapshot();
    })
})