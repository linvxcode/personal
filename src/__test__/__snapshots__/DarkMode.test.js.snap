// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Darkmode Render Darkmode 1`] = `undefined`;
